app/utils/stream.ts

```
@@ -0,0 +1,96 @@
+      body = [],
+    } = options || {};
+    let unlisten: Function | undefined;
+    let request_id = 0;
+    const ts = new TransformStream();
+    const writer = ts.writable.getWriter();
+
+    let closed = false;
+    const close = () => {
+      if (closed) return;
+      closed = true;
+      unlisten && unlisten();
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)